server/reports/encounters/2_6/convert_data_js/src/utils.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+  applyDaysLate,
+  getStatus,
+};
```

Comment:
Exports at the top this is MADNESS??

Reply:
am i wrong for wanting to open a file and know what leaves it 😢